tools/build/lib/byond.js

```diff
@@ -58,15 +63,69 @@ const getDmPath = async () => {
   return dmPath;
 };
 
+
+
+const getNamedByondVersionPath = (namedVersion) =>{
+  const all_entries = getMapFileContents(true)
+  const map_entry = all_entries.find(x => x.name === namedVersion);
+  if(map_entry === undefined){
+    Juke.logger.error(`No named byond version with name "${namedVersion}" found.`);
+    throw new Juke.ExitCode(1);
+  }
+  return map_entry.path;
+}
+
+const getDefaultNamedByondVersionPath = () =>{
+  const all_entries = getMapFileContents(false)
+  const map_entry = all_entries.find(x => x.default == true);
+  if(map_entry === undefined)
+    return []
+  return [map_entry.path];
+}
+
+
+/** @type {[{ name, path, default }]} */
+let mapFileContents;
+export const NamedVersionFile = "tools/build/dm_versions.json"
+
+const getMapFileContents = (throw_on_fail) => {
```

Comment:
Confusing name, I thought this has to do with byond maps, but this is essentially just a version list.

Reply:
Sure, will rename this